pkg/util/logger/logger.go

```diff
@@ -9,10 +9,10 @@ import (
 
 // Logger represents a component
 // for writing messages to log.
-//
-// It is a type alias of
-// go.uber.org/zap.Logger.
-type Logger = zap.Logger
+type Logger struct {
+	*zap.Logger
```

Comment:
IMO it doesn't make any sense to do such wrapper. I think we should either use `zap.Logger` or completely autonomous `Logger` instance. The latter should be a pure mediator encapsulating `zap.Logger.`

Reply:
> IMO it doesn't make any sense

yes, agree. but i do not want to bring refactoring as long as it is possible, see comment below